civis/tests/test_parallel.py

```diff
@@ -416,17 +418,16 @@ def test_result_exception_no_result():
     # a generic TransportableException back.
     callback = mock.MagicMock()
 
-    # Passing the client mock as an argument instead of globally
-    # patching the client tests that the _CivisBackendResult
-    # uses the client object on the input CivisFuture.
-    mock_client = create_client_mock()
-    mock_client.scripts.list_containers_runs_outputs.return_value = []
+    mock_client = create_client_container_mock(1, 2, state='failed',
+                                               run_outputs=[])
     fut = ContainerFuture(1, 2, client=mock_client)
-    fut._set_api_exception(Response({'state': 'failed'}))
     res = civis.parallel._CivisBackendResult(fut, callback)
+    fut._set_api_exception(CivisJobFailure(
```

Comment:
h/t @stephen-hoover for noticing that the arguments to `_set_api_exception` were not actually the expected order/type here (although the test happened to pass anyway).